run

```diff
@@ -499,6 +499,14 @@ rpath() {
     cut -d: -f2 <<< "$1"
 }
 
+load_loop_module() {
```

Comment:
My intention is to have a piece of code that works for both platforms like the rest of ktest.

Reply:
So, I would love to see the function gone.